server/JsDbg.Gdb/Makefile

```diff
@@ -0,0 +1,28 @@
+PREFIX=/usr/local
+DOTNET=$(HOME)/dotnet/dotnet
+
+PUBLISH=bin/Release/netcoreapp2.1/linux-x64/publish
```

Comment:
Is `netcoreapp2.1` a constant, or should we get it from the version somehow? Looks like the current download page points to 2.2 - does it still produce output at netcoreapp2.1?

Reply:
Looks like this matches the existing path from the python script, so I'm guessing not an issue (or at least can be tackled in a separate change).

Reply:
Fairly sure this comes from the TargetFramework in the .csproj file -- I do have the dotnet sdk 2.2 and it does output into netcoreapp2.1.